.travis.yml

```diff
@@ -0,0 +1,105 @@
+# Travis CI (MIT License) configuration file for Options Importer
+# @link https://travis-ci.org/
+
+# Declare project language.
+# @link http://about.travis-ci.org/docs/user/languages/php/
+language: php
+
+# Specify when Travis should build.
+branches:
+  only:
+    - master
+
+cache:
+  directories:
+    - $HOME/.composer/cache
+    - ./vendor
+
+matrix:
+  include:
+    - php: '5.3'
+      env: WP_VERSION=4.7.6
+      dist: precise
+    - php: '5.3'
+      env: WP_VERSION=latest
+      dist: precise
+    - php: '5.6'
+      env: WP_VERSION=latest
+    - php: '5.6'
+      env: WP_TRAVISCI=phpcs
+    - php: '7.0'
+      env: WP_VERSION=latest
+    - php: '7.1'
+      env: WP_VERSION=latest
+    - php: '7.2'
+      env: WP_VERSION=latest
+    - php: '7.3'
+      env: WP_VERSION=latest
+    - php: '7.3'
+      env: WP_VERSION=nightly
+  fast_finish: true
```

Comment:
Could PHP 7.4 be tested as well, since that it released?

And the nightly release of PHP 8 (but allowed to fail)?

Reply:
I agree that adding 7.4 makes sense